crates/rune-macros/src/context.rs

```diff
@@ -336,6 +378,67 @@ impl Context {
                             }
                         },
                     });
+                } else if meta.path == GET_REF {
```

Comment:
When dealing with nested lifetimes (i.e. an `AnyRef` with an `&AnyRef` field), I needed a way to differentiate from a regular `Any`. But this could possibly be cleaned up.